### PR TITLE
added scroll bar to filter options in desktop mode only

### LINKED ISSUE #689: Ability to Scroll in Sidebar
## Problem Description
In the Desktop view, you cant scroll the sidebar. For example, the "All Sources" link has many options in it. To access the bottom options you should scroll the entire page. Also, the sidebar scrolls down when you scroll to the bottom of the page.

## Solution Description
Adding a separate scroll option to the sidebar solves both the issues

## Alternatives
None

## Additional context
If this issue is considered I would like to work on it

### PR DESCRIPTION
**Pull request by satyaom**
Fixes #689 

added a scroll bar only in desktop mode.

****Desktop View** (with scroll bar)**

Peek 2020-03-15 03-45
Peek 2020-03-15 03-47
Peek 2020-03-15 03-56
Peek 2020-03-16 00-24

****Mobile View** (without scroll bar)**

Peek 2020-03-15 04-00
Peek 2020-03-15 04-001

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,5 +122,19 @@
     "> 1%",
     "last 2 versions",
     "not ie <= 8"
-  ]
+  ],
```
Comment (akmadian) [inline package.json]: Please remove all additions in package.json except the additions of the dependencies.

Comment (satyaom) [reply package.json]: Ok, will remove all additions

Comment (satyaom) [reply package.json]: I also love to contribute in common creative org

Comment (satyaom) [reply package.json]: what you going to do about simplebar package, does @brenoferreira  approve it.

Comment (satyaom) [reply package.json]: If @brenoferreira approve please merge it.

Comment (satyaom) [reply package.json]: if you want any change please inform me.

**Comment by brenoferreira** [comment]
@satyaom any updates on this?

**Comment by brenoferreira** [comment]
Marking as stale and closing

Feel free to reopen if you want to get back to this

**Pull request closed without merging**